tools/benchmark/CHANGELOG.md

```diff
@@ -8,8 +8,7 @@
 
 ## 0.49.0
 
-Provide @benchmarkCustom feature to log custom measurements. To profile custom usage, define tests using the `benchmarkCustom()` function. The argument `run` to the function includes a reporter with `addMeasurement()`
-to write custom data to report.
+Provides @CustomBenchmark feature to log custom measurements. To profile custom usage, define tests using the `benchmarkCustom()` function. The argument `run` to the function includes a reporter with `addMeasurement()` to write custom data to report.
```

Comment:
```suggestion
Adds a feature to run benchmarks that log custom measurements. To use it, define tests using the `benchmarkCustom()` function. The `run` argument passed to the function is itself a function that will get passed a reporter object with an `addMeasurement()` method, that the test code can use to define custom data to report as the benchmark output. These custom-measurement benchmarks get the string `@CustomBenchmark` appended to their title, so they can be run selectively using `mocha`'s `fgrep` flag.
```

Reply:
Nit: I also always recommend line-breaking along sentence boundaries. Makes future edits/diffs easier to review. :)